packages/cdk/lib/construct/web.ts

```diff
@@ -273,7 +274,7 @@ export class Web extends Construct {
         VITE_APP_SAML_COGNITO_DOMAIN_NAME: props.samlCognitoDomainName ?? '',
         VITE_APP_SAML_COGNITO_FEDERATED_IDENTITY_PROVIDER_NAME:
           props.samlCognitoFederatedIdentityProviderName ?? '',
-        VITE_APP_AGENT_NAMES: JSON.stringify(props.agentNames),
+        VITE_APP_AGENTS: JSON.stringify(props.agents),
```

Comment:
```suggestion
        VITE_APP_AGENTS: JSON.stringify(props.agents),
        VITE_APP_MCP_SERVERS_CONFIG: props.mcpServersConfig ?? '{}',
```

このような感じで mcp.json から取得した mcp サーバーの設定情報を環境変数に埋め込む必要がありそうでした。

Reply:
ありがとうございます。こちら修正しました